src/renderer/components/kube-object-status-icon/kube-object-status-icon.tsx

```diff
@@ -2,105 +2,106 @@ import "./kube-object-status-icon.scss";
 
 import React from "react";
 import { Icon } from "../icon";
-import { KubeObject } from "../../api/kube-object";
 import { cssNames, formatDuration } from "../../utils";
-import { KubeObjectStatusRegistration, kubeObjectStatusRegistry } from "../../../extensions/registries/kube-object-status-registry";
-import { KubeObjectStatus, KubeObjectStatusLevel } from "../../..//extensions/renderer-api/k8s-api";
-import { computed } from "mobx";
+import { KubeObject, KubeObjectStatus, KubeObjectStatusLevel } from "../../..//extensions/renderer-api/k8s-api";
+import { kubeObjectStatusRegistry } from "../../../extensions/registries";
 
-interface Props {
+export interface ObjectProps {
```

Comment:
Why renamed to `ObjectProps` and exported? I see that the only component is using it placed inside same file.
About new name: maybe it's just me, but it started to direct my mind to native js objects at first. Maybe no need to rename?

Reply:
I really don't remember why I did this. Will revert.